src/authorization/routes.authorization.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
```

Comment:
Finally good name for the file. Let's rename the other files using the same naming pattern